src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -133,21 +158,24 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         }
 
         /// <summary>
-        /// Maps key types or key values into a floating point vector.
+        /// Create a <see cref="KeyToVectorMappingEstimator"/>, which maps key types or key values into a floating point vector.
```

Comment:
>which maps key types or key values into a floating point vector. [](start = 64, length = 64)

which maps the value of <see cref="KeyDataViewType"/> into a floating point vector representing the value.